INSTALL.md

```diff
@@ -55,7 +55,7 @@ cmake -DFDEEP_BUILD_UNITTEST=ON ..
 make unittest
 cd ../..
 ```
-
+The Unit Tests require Python3, please make sure Python3 has been installed rightly.
```

Comment:
A very minor thing, but let's use "correctly" instead of "rightly".

Reply:
Or, let's even make it terser:

"The unit tests require Python3 to be installed."